src/components/ContactGroups/SelectBox/Control.jsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+
+import Button from 'cozy-ui/transpiled/react/Button'
+import Icon from 'cozy-ui/transpiled/react/Icon'
+import palette from 'cozy-ui/transpiled/react/palette'
+import { useI18n } from 'cozy-ui/transpiled/react/I18n'
+
+const Control = ({ selectProps: { toggleMenuIsOpen } }) => {
+  const { t } = useI18n()
+  const toggleMenu = () => {
+    toggleMenuIsOpen()
+  }
```

Comment:
We should useCallback() to not recreate this method at every render

Reply:
this func is removed few commits later and passed in props